kong/llm/drivers/azure.lua

```diff
@@ -92,16 +104,21 @@ function _M.configure_request(conf)
     local url = fmt(
       "%s%s",
       ai_shared.upstream_url_format[DRIVER_NAME]:format(conf.model.options.azure_instance, conf.model.options.azure_deployment_id),
-      ai_shared.operation_map[DRIVER_NAME][conf.route_type].path
+          conf.model.options
+      and conf.model.options.upstream_path
+      or ai_shared.operation_map[DRIVER_NAME][conf.route_type].path
     )
     parsed_url = socket_url.parse(url)
   end
 
+  if string.sub(parsed_url.path, 1, 1) ~= "/" then
+    parsed_url.path = "/" .. parsed_url.path
+  end
```

Comment:
A common function to prepend a slash to relative paths should be introduced and used.

Reply:
This PR was re-done, due to messy rebases from other features.

See: https://github.com/Kong/kong/pull/12903/files#r1575189707